lib/crawlers/blockHarvester.js

```diff
@@ -54,8 +54,11 @@ module.exports = {
     //
     console.log(`[PolkaStats backend v3] - Block harvester - \x1b[32mAdded ${addedBlocks} blocks in ${((endTime - startTime) / 1000).toFixed(0)}s\x1b[0m`);
     console.log(`[PolkaStats backend v3] - Block harvester - \x1b[32mNext execution in 60m...\x1b[0m`);
-    setTimeout(module.exports.blockHarvester(api, pool, config), POLLING_TIME);
-    
+
+    setTimeout(
+      () => module.exports.blockHarvester(api, pool, config),
+      config.pollingTime,
+    );
```

Comment:
The problem is that merging this to develop right now would break things... as `pollingTime` in `config` only exists currently in #69

Reply:
Ok, I flag this PR as [WIP] until #69 is merged

Reply:
Perfect